monitor/service.go

```diff
@@ -200,6 +196,12 @@ func (m *Monitor) Statistics(tags map[string]string) ([]*Statistic, error) {
 					}
 					statistic.Values[kv.Key] = f
 				})
+			case "references":
+				s := subKV.Value.(*expvar.Int).String()
+				count, _ := strconv.ParseInt(s, 10, 32)
+				if count < 2 {
+					influxdb.DeleteStatistics(kv.Key)
```

Comment:
Same question here: Why does reading the stats indicate time to delete them? Is it so that they get reported one last time before being deleted?